### ISSUE TITLE
fresh install does not work if there is an existing fos installation

### ISSUE THREAD

**Issue by cgheonea** [zgelici/FOS-Streaming-v1#38]
Please do not use fresh_install script if there is an existing installation of fos into your machine.
Do a format first of all before anything.
The recommended version of Linux is: Ubuntu 14.05/15.04 64-bit.

Issue ticket to be updated once fixed.

**Comment by r15ch13** [comment]
Why is your setup so confusing and destructive? (like writing the nginx conf if the port is changed in the webinterface)
How about splitting it into seperate sections per case. (and into a subdirectory! this project is a bit of a mess)
Also try to avoid hardcoded path settings, so it maybe possible to install it on an existing webserver.
Here is my attempt to run a previous version via Docker and Vagrant. https://github.com/r15ch13/fos-streaming

**Comment by zgelici** [comment]
Hello r15ch13m,

You are right, its confused that the installation is confusing, we trying to make this better. 

Also we did something wrong with pushing changes to the master, but i revert it back to the stable one. 
I created a develop branch. This branch is for development.  We trying to make it dynamic, but we need to make it easy.

We are also trying to fix the structure of FOS-Streaming. 

This is very nice: https://github.com/r15ch13/fos-streaming

Good job.
